test/e2e/specs/settings.spec.ts

```diff
@@ -0,0 +1,217 @@
+import { browser } from '@wdio/globals'
+import type { OutputView, Setting, SettingsEditor, Workbench } from 'wdio-vscode-service'
+import isEqual from 'lodash/isEqual'
+import { Key } from 'webdriverio'
+import { $ } from 'zx'
+import { getFirstWelcomeViewText } from '../helpers/queries'
+import {
+  expectOutputToContain,
+  expectStandardSidebarViewsToBeVisible,
+} from '../helpers/assertions'
+import {
+  clearInput,
+  closeRadicleViewContainer,
+  openRadicleViewContainer,
+} from '../helpers/actions'
+import { nodeHomePath } from '../constants/config'
+
+const tempNodeHomePath = `${nodeHomePath}.temp`
+
+describe('Settings', () => {
+  let workbench: Workbench
+  let settings: SettingsEditor
+
+  before(async () => {
+    workbench = await browser.getWorkbench()
+    settings = await workbench.openSettings()
+    await openRadicleViewContainer(workbench)
+    await expectStandardSidebarViewsToBeVisible(workbench)
+  })
+
+  after(async () => {
+    await closeRadicleViewContainer(workbench)
+  })
+
+  describe('VS Code, when updating the "Path to Rad Binary" setting,', () => {
+    let pathToRadBinarySetting: Setting
+
+    before(async () => {
+      pathToRadBinarySetting = await settings.findSetting(
+        'Path To Rad Binary',
+        'Radicle',
+        'Advanced',
+      )
+    })
+
+    after(async () => {
+      await clearInput(await getSettingsSearchBox(settings))
+    })
+
+    afterEach(async () => {
+      await clearTextSetting(pathToRadBinarySetting)
+
+      await expectStandardSidebarViewsToBeVisible(workbench)
+    })
+
+    it('warns the user if the rad binary is not found', async () => {
+      await setTextSettingValue(pathToRadBinarySetting, '/tmp')
+
+      await expectRadBinaryNotFoundToBeVisible(workbench)
+    })
+
+    it('recognizes the rad binary when a valid path is specified', async () => {
+      await $`cp -r ${nodeHomePath} ${tempNodeHomePath}`
+
+      await setTextSettingValue(pathToRadBinarySetting, `/tmp`)
+
+      await expectRadBinaryNotFoundToBeVisible(workbench)
+
+      await setTextSettingValue(pathToRadBinarySetting, `${tempNodeHomePath}/bin/rad`)
+
+      await expectStandardSidebarViewsToBeVisible(workbench)
+
+      await $`rm -rf ${tempNodeHomePath}`
+    })
+
+    // This functionality does not seem to work
+    // eslint-disable-next-line max-len
+    it.skip('recognizes if the directory is created *after* the setting is updated', async () => {
```

Comment:
```suggestion
    it.skip('recognizes if the directory is created *after* the setting is set', async () => {
```

Minor copy improvement. At least for my brain, makes the case description a tad easier to parse.

Reply:
Done!